src/Components/WebAssembly/testassets/WasmLinkerTest/WasmLinkerTest.csproj

```diff
@@ -3,6 +3,7 @@
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
     <SelfContained>true</SelfContained>
+    <UseMonoRuntime>true</UseMonoRuntime>
```

Comment:
Is this really related to this PR, or just something we need for another reason?

Could you clarify why we need it (and didn't before)?

Reply:
This change was ported from the PR in main: https://github.com/dotnet/aspnetcore/pull/31748/files. The targeting packs in the runtime were recently renamed and this is now required if you do not reference the BlazorWebAssembly SDK (which this project does not)